<ui version="4.0">
 <class>TGo4ConnectServer</class>
 <widget class="QDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to server analysis</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ServerLbl">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Connection</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ConnectionCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Go4 sockets</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HTTP server</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="TextLabel2_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Host</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ClientNode">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>hostname of analysis server</string>
       </property>
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel2_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="PortNumber">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Port number of analysis server to connect. Only for gui in client mode.</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>5000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ModeLbl">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Account</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AccountEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>account name at web server</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ModeCombo">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Login account </string>
       </property>
       <item>
        <property name="text">
         <string>Observer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Controller</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Administrator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>other...</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel4">
       <property name="text">
        <string>Password</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PasswordEdit">
       <property name="toolTip">
        <string>password</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="DefaultPassCheck">
       <property name="toolTip">
        <string>if selected use Go4 default passwords for login role</string>
       </property>
       <property name="text">
        <string>default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer257">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>210</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ConnectBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="CloseBtn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/close16.png</normaloff>:/icons/close16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
</ui>
